.pre-commit-config.yaml

```diff
@@ -39,25 +40,24 @@ repos:
     rev: v3.15.2
     hooks:
     -   id: pyupgrade
-        args: [--py37-plus]
+        args: [--py39-plus]
```

Comment:
Also here

```suggestion
        args: [--py310-plus]
```

Reply:
@domfournier I have committed changes including the requested line which removes support for python 3.9. Changes pass the pre-commit checks and tests are still passing in python 3.12